src/reference/faq.rst

```diff
@@ -9,6 +9,62 @@ Installation
 
 There are many ways to install Nextstrain, and we aim to simplify the installation guide so it is easy to follow along. Here, you will find answers to some common questions about the installation process.
 
+
+.. _what-are-docker-conda-mamba-wsl-etc:
+
+What are Docker, Conda, Mamba, WSL, etc.?
+-----------------------------------------
+
+`Docker <https://docker.com/>`_ is a container system available for all platforms.
+When you use Docker to run Nextstrain components, you don’t need to manage any other Nextstrain software dependencies as validated versions are already bundled into `a container image by the Nextstrain team <https://github.com/nextstrain/docker-base/>`_.
+
+`Conda <https://docs.conda.io/en/latest/>`_ is a package and environment management system that allows you to install Python and other software into controlled environments without disrupting other software you have installed (e.g., on your computer, your shared cluster, etc.).
+Miniconda is the minimal installation of the ``conda`` command, the command-line interface to Conda.
+
+`Mamba <https://github.com/mamba-org/mamba>`_ is a drop-in replacement for most ``conda`` functionality that provides faster installation times, especially for complex environments like the one Nextstrain requires.
+
+`WSL <https://docs.microsoft.com/en-us/windows/wsl/about>`__ is Windows Subsystem for Linux, a full Linux environment integrated into Windows without the need for awkward virtual machines.
+Nextstrain's installation guide works with WSL 2 but not WSL 1.
+
+
+.. _choosing-a-runtime:
+
+Should I choose the Docker or native runtime?
+---------------------------------------------
+
+The two runtimes provide the same experience running Nextstrain workflows with ``nextstrain build`` (and most other ``nextstrain`` commands) but vary in installation and update methods and predictability/stability over time.
+
+There's not one right answer for everyone or every situation.
+That's why we provide both options (and will potentially provide more in the future).
+The Nextstrain team uses both runtimes extensively.
+
+Your preference is perhaps the best reason to choose one vs. the other.
+
+   - If you have a preference for containers, then choose our Docker runtime.
+   - If you have a preference for Conda, then choose our native runtime.
```

Comment:
Refer to runtimes consistently:

```suggestion
   - If you have a preference for containers, then choose the Docker runtime.
   - If you have a preference for Conda, then choose the native runtime.
```

Reply:
Committed as 21b65c3. I can fix it up into the first commit in this PR if you'd like.

Reply:
Sure, that sounds reasonable.